src/fetch_orderbook.ts

```diff
@@ -0,0 +1,86 @@
+import Web3 from "web3";
+import { AbiItem } from "web3-utils";
+import { decodeOrdersBN, OrderBN } from "@gnosis.pm/dex-contracts/src/index";
+import { Orderbook, Offer } from "@gnosis.pm/dex-contracts/src/orderbook";
+import { Fraction } from "@gnosis.pm/dex-contracts/src/fraction";
+import { BatchExchangeViewer } from "@gnosis.pm/dex-contracts/build/types/BatchExchangeViewer";
+import batchExchangeViewerAbi from "@gnosis.pm/dex-contracts/build/contracts/BatchExchangeViewer.json";
+
+let lastBatchId = 0;
+let lastOrderbooks: Map<string, Orderbook> = null;
+
+const web3 = new Web3("https://node.mainnet.gnosisdev.com");
+const batchExchangeViewer = new web3.eth.Contract(
+  batchExchangeViewerAbi.abi as AbiItem[],
+  batchExchangeViewerAbi.networks[1].address
+) as BatchExchangeViewer;
+
+export async function fetchOrderbook() {
+  const block = await web3.eth.getBlock("latest");
+  const currentBatchId = batchIdFromTimestamp(block.timestamp as number);
+  if (currentBatchId > lastBatchId) {
+    lastBatchId = currentBatchId;
+    // tslint:disable-next-line:no-console
+    console.log(`Fetch orderbook for batch: ${currentBatchId}`);
```

Comment:
If possible, use dex-js logger

Reply:
Any reason you implemented your own logger over using e.g. `typescript-logging` ?

Reply:
We didn't implement a logger, we just a logging library (debug)